cli/azd/extensions/microsoft.azd.ai.builder/internal/cmd/root.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func NewRootCommand() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:           "azd ai builder <command> [options]",
```

Comment:
```suggestion
		Use:           "azd ai <command> [options]",
```

Also, where is this displayed? I don't see it when I run `azd ai` or `azd ai --help`:

![image](https://github.com/user-attachments/assets/a4bd5139-7238-4876-90f6-841751adf236)